### ISSUE TITLE
The UI bugs out when there are too many files

### ISSUE THREAD

**Issue by bebraw** [christianalfoni/webpack-bin#128]
The UI starts cropping when you have so many files that they go to two rows. Here's a [demo](http://www.webpackbin.com/N1XA3ETlZ). Screenshot below:

![screenshot 2016-04-30 10 26 06](https://cloud.githubusercontent.com/assets/166921/14934595/ff4d7a66-0ebd-11e6-9d13-532c0b29fc89.png)

**Comment by christianalfoni** [comment]
Yeah, responsive crap, hehe... next version will get some responsive handling. Its just quite a bit of work so not looking forward to it, but getting some help of a colleague to fix this stuff :-) Will be fixed in next version
